.github/workflows/assets/minio.yaml

```
@@ -0,0 +1,46 @@
+metadata:
+  name: my-minio-fs
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: my-minio-fs
+    spec:
+      volumes:
+      - name: data
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)